test/helpers/form-monkey.ts

```diff
@@ -19,19 +18,18 @@ interface IUIInput {
  * helper for simulating user input into SLOBS forms
  */
 export class FormMonkey {
-  private client: SpectronClient;
-
-  constructor(private t: TExecutionContext, private showLogs = false) {
-    this.client = t.context.app.client;
+  constructor(
+    private t: TExecutionContext,
+    private formSelector = 'body',
```

Comment:
Can we make `formSelector` the last argument? I agree that it could be more intuitive this way, but it is a breaking change for tests, which wouldn't be if the took that last.

Reply:
This PR already introduced a breaking change for tests. But there are still no so many places affected by this breaking change so I fixed all of them.